src/integration-test/java/com/commercetools/sync/integration/commons/utils/ProductTypeITUtils.java

```diff
@@ -529,5 +511,17 @@ private static void assertLocalizedEnumsValuesAreEqual(
             });
   }
 
+  @Nonnull
+  public static Attribute createNestedAttributeValueReferences(
+      @Nonnull final String attributeName, @Nonnull final Object referenceValue) {
+    return AttributeBuilder.of().name(attributeName).value(referenceValue).build();
```

Comment:
I see you changed [in this PR](https://github.com/commercetools/commercetools-sync-java/pull/1083/commits/ce0189aabe11d7bf2facb8659528772ab2aa001a) from Map to AttributeBuilder. Is it the only way we support? If so, we should document this.

Reply:
Ok, i found an issue with this change. So, i have to make it work for Map too.

Reply:
If it's too hard to support both ways, you could also document this as a restriction and support only AttributeBuilder.